src/DotNetWorker.Grpc/Definition/GrpcFunctionDefinition.cs

```diff
@@ -69,5 +74,14 @@ public GrpcFunctionDefinition(FunctionLoadRequest loadRequest, IMethodInfoLocato
 
             return ImmutableDictionary<string, object>.Empty;
         }
+
+        private bool ParametersContainCancellationToken(ImmutableArray<FunctionParameter> parameters)
+        {
+            if (parameters.Any(p => p.Type == typeof(CancellationToken) || p.Type == typeof(CancellationToken?)))
```

Comment:
I think you can remove the if/else statement and just return the `.Any` result directly.

``` CSharp
return parameters.Any(p => p.Type == typeof(CancellationToken) || p.Type == typeof(CancellationToken?));
```